pkg/operation/botanist/controlplane/kube_apiserver_service.go

```diff
@@ -29,6 +29,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
one line above imports the same package, remove this one and just use `client.<...>`